.github/workflows/rust.yml

```
@@ -68,11 +68,11 @@ jobs:
         uses: actions/cache@v2
```

Not sure if the scope of this PR is just for pinning `grcov` but we can also update the cache action (can be done in a different PR)

```suggestion
        uses: actions/cache@v2.1.5
```

---

Thanks for that! Yeah, I don't see why we couldn't include those changes :)

---

just for reference: https://github.com/actions/cache/releases if we want to check release notes.